cookbooks/cdo-github-access/metadata.rb

```diff
@@ -4,4 +4,7 @@
 license          'All rights reserved'
 description      'Installs/Configures cdo-github-access'
 long_description File.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.1.10'
+version          '0.1.11'
```

Comment:
Note you don't have to update this yourself; these version numbers are determined programmatically, and [updates are committed as part of the staging build](https://github.com/code-dot-org/code-dot-org/commit/661abe389f6fc082e67a3e70fe1d962e85c87ed4).

That being said, I don't think it'll hurt anything to update them manually. Just want to make sure you know!

Reply:
oh interesting, I couldn't get my local tests to pass until I upped this version, so maybe it was just a local issue